config/config.go

```diff
@@ -244,6 +256,13 @@ func (s *SourceSubnet) Validate() error {
 		}
 	}
 
+	// Validate the allowed destinations
```

Comment:
We should also validate that the allowed destinations are all present in `cfg.DestinationSubnets`

Reply:
checked [here](https://github.com/ava-labs/awm-relayer/pull/59/files#diff-fe44f09c4d5977b5f5eaea29170b6a0748819c9d02271746a20d81a5f3efca17R191-R201)